src/wrapper/simulation_objects.py

```diff
@@ -84,3 +86,222 @@ def update(self, data: dict):
 
     def add_subscriptions(self) -> int:
         return 0
+
+
+class Switch(Node):
+    """A switch in the simulation which can point either left or right
+    (see `ISimulationSwitch.State`)
+    """
+
+    class State(Enum):
+        """The possible states of the switch"""
+
+        LEFT = 1
+        RIGHT = 2
+
+    state: State
+
+
+class Track(SimulationObject):
+    """A track in the simulation where trains can drive along"""
+
+    blocked: bool
+    speed_limit: float
+
+
+class Platform(SimulationObject):
+    """A platform where trains can arrive, load and unload passengers and depart"""
+
+    track: Track
+    station_name: str
+    platform_number: int
+    blocked: bool
+
+
+class Train(SimulationObject):
+    """A train driving around in the simulation."""
+
+    # pylint: disable=too-many-instance-attributes
```

Comment:
Maybe it is possible to move some grouped instance attributes into a helper class, so that this is not necessary anymore, and it is easier to understand quickly? 
I thought about a class TrainType, which has the fields `_max_speed`, `_priority` and `_train_type`.